tests/sentry/test_wsgi.py

```diff
@@ -0,0 +1,32 @@
+import inspect
+import subprocess
+import sys
+
+
+def subprocess_test_wsgi_warmup():
+    # isort: off
+    import sentry.wsgi  # noqa
+    import sys
+
+    assert "django.urls.resolvers" in sys.modules
+
+    import django.urls.resolvers
+
+    resolver = django.urls.resolvers.get_resolver()
+    assert resolver._populated is True  # type: ignore[attr-defined]
```

Comment:
- I would just write this as a string personally -- about equivalent to turning off the linters (and you don't have to use `inspect.getsource` which usually sets off alarm bells due to being a perf problem)
- this should assert that the sentry urls module is in sys.modules (and perhaps that it's _not_ in sys.modules before importing `sentry.wsgi`)